tools/lint/README.md

```diff
@@ -0,0 +1,131 @@
+## Introduction
+
+We have a lint tool for catching common mistakes in test files. Now it can
+check html, htm, xhtml, xhtm, json and xml files.
+- You can run it manually by starting the `lint.py` executable from the root of your local
+crosswalk-test-suite working directory like this:
+
+```
+./tools/lint/lint.py
```

Comment:
You'd better add "-f" here, as a separated section to introduce checking all test suite.